Packs/SecneurXAnalysis/TestPlaybooks/Detonate_URL_-_SecneurX_Anlaysis_-_Test.yml

```
@@ -0,0 +1,180 @@
+id: b1c38546-35cb-4006-8085-58e4f501e907
```

Please change the value to the value of `name`
**Both values should be the same**

```suggestion
id: Detonate URL - SecneurX Anlaysis - Test
```